thomas/index.html

```diff
@@ -51,7 +55,26 @@
         function displayFish() { 
             fishQuantity.innerHTML = fish; 
         }
+	function add fps(){
+	fish = fish + fps
+	}
+	
+	    
+	    function buyapond() {
+	    if(fish>19)
+		    fish = fish - 20
+		    fps = fps + 1
+	    }
+		    
+		    
+	    
```

Comment:
You need a `setInterval` somewhere to call your `addfps` function:
`setInterval(addfps, 1000)`